crates/cxx-qt-build/Cargo.toml

```diff
@@ -26,6 +25,6 @@ version_check = "0.9"
 
 [features]
 default = ["qt_gui", "qt_qml"]
-qt_gui = ["cxx-qt-lib-headers/qt_gui"]
-qt_qml = ["cxx-qt-lib-headers/qt_qml"]
+qt_gui = []
```

Comment:
we still need to set the defines in cxx-qt-build, wonder if instead of `with_headers(prefix, files)` it should also have defines or something ? Then have `with_opts(CxxQtBuilderOpts)` and have `cxx_qt_lib` return a struct, but then we have a circular dependency ...

Reply:
Done as Opts

Reply:
Then why do we still need this feature in cxx-qt-build?

Reply:
Because `cxx-qt-build` also has the following which defaults to enabling the `gui` and `qml`  Qt modules in the Qt builders.

```rust
        let mut qt_modules = HashSet::new();
        qt_modules.insert("Core".to_owned());
        #[cfg(feature = "qt_gui")]
        qt_modules.insert("Gui".to_owned());
        #[cfg(feature = "qt_qml")]
        qt_modules.insert("Qml".to_owned());
```

Wonder how we should handle those, should our `CxxQtBuilderOpts` have the ability to pass in Qt modules that are required too ? Probably yes as that would be quite useful if you made a Bluetooth library to be able to then say i need the Bluetooth module as well.